src/status_im/node/core.cljs

```diff
@@ -60,6 +60,15 @@
     (assoc :ListenAddr ":30304"
            :DataDir (str (:DataDir config) "_dev"))))
 
+(def login-node-config
+  {:WalletConfig (cond-> {:Enabled true}
+                   (not= config/opensea-api-key "")
```

Comment:
and then it would be just `(not (nil? config/opensea-api-key))`

Reply:
in that case it could be just `nil` we don't need condition, but not sure how it will work in json and then in status-go